.github/workflows/windows.yml

```diff
@@ -10,6 +10,9 @@ permissions:
 jobs:
   pytest-coverage: # from pytest_coverage.yml
     runs-on: windows-latest
+    env:
+      # A workaround for long FQDN names provided by GitHub actions.
+      FQDN: "localhost"
```

Comment:
I suggest making this a global env setting for all jobs in the `windows.yml` workflow.

Reply:
Done.